jobs/sentry-error-loader/sentry_error_loader.py

```diff
@@ -45,6 +45,7 @@ def fetch_and_clean_from_clickhouse(project_slug, target_date):
         dt=target_date,
         execution_ts=pendulum.now(),
         filename="events.jsonl.gz",
+        data=pd.DataFrame(),
```

Comment:
is this necessary? You can probably just check `is not None` later

Reply:
Addressed in [8650185](https://github.com/cal-itp/data-infra/pull/2356/commits/86501859f71672b832bbab6f639a22064c08479d)